trace-dispatcher/src/Cardano/Logging/Tracer/Composed.hs

```diff
@@ -119,3 +126,32 @@ mkCardanoTracer' trStdout trForward mbTrEkg name namesFor severityFor privacyFor
         case mbEkgTrace <> mbForwardTrace <> mbStdoutTrace of
           Nothing -> pure $ Trace NT.nullTracer
           Just tr -> pure (preFormatted backends tr)
+
+-- A simple dataPointTracer which supports building a namespace and entering a hook
+-- function.
+mkDataPointTracer :: forall dp. ToJSON dp
+  => Trace IO DataPoint
+  -> (dp -> [Text])
+  -> IO (Trace IO dp)
+mkDataPointTracer trDataPoint namesFor = do
+    let tr = NT.contramap DataPoint trDataPoint
+    pure $ withNamesAppended namesFor tr
```

Comment:
2 spaces, please.

Reply:
I read in the Consensus formatting document,  that 4 spaces are an suggested option, because it makes adding of where clauses easy.

Reply:
Ok, but there is a style guide for this repository, and I see 2 spaces here: https://github.com/input-output-hk/cardano-node/blob/master/STYLE.md#indentation

Reply:
Yes, there is a conflict there!

I generally prefer consensus style, in most cases.

Sadly, the node has chosen to follow a different style.

And we must abide.